source/OutfitterPanel.cpp

```diff
@@ -168,6 +168,9 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 
 	const Font &font = FontSet::Get(14);
 	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &highlight = *GameData::Colors().Get("outfitter difference highlight");
+	bool highlightDifferences = false;
+
```

Comment:
If there's going to be an empty line here, I think the `bool highlightDifferences` ought to be separated from the above since those are all constants used here to provide more convenient access to objects stored elsewhere, while the bool is a local that will be updated in the following block.
```suggestion

	bool highlightDifferences = false;

```
The empty line between the bool declaration and the following code is probably not necessary if there's an empty line before it, but I won't say no if you really want it.
```suggestion

	bool highlightDifferences = false;
```